infra/cifuzz/filestore/gitlab/__init__.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2021 Google LLC
```

Comment:
Can you add some tests?

Reply:
@jonathanmetzman what kind of tests do you think about ?
I guess the tests with value would be end to end tests, not unit tests here, so as to check that we use Gitlab API correctly to upload/download crashes/coverage/builds/corpus